Fix changelog bug and add entry for PR #5538

halfalicious opened the pull request:
PR 5538 removed the --private option from aleth

chfast commented on CHANGELOG.md:
```diff
@@ -13,5 +13,6 @@
 - Fixed: [#5519](https://github.com/ethereum/aleth/pull/5519) Correctly handle Discovery messages with known public key but unknown endpoint.
 - Fixed: [#5523](https://github.com/ethereum/aleth/pull/5523) [#5533](https://github.com/ethereum/aleth/pull/5533) Fix syncing terminating prematurely because of race condition.
 - Added: [#5526](https://github.com/ethereum/aleth/pull/5526) Improved logging when loading chain config json containing syntax error.
+- Added: [#5538](https://github.com/ethereum/aleth/pull/5538) Removed --private flag from aleth command-line arguments.
```
This should be `Removed:`. See https://keepachangelog.com/en/1.0.0/

chfast commented on CHANGELOG.md:
```diff
@@ -2,7 +2,7 @@
 ## [1.6.0] - Unreleased
 
-- Added: [#5484](https://github.com/ethereum/aleth/pull/5485) aleth-bootnode now by default connects to official Ethereum bootnodes. This can be disabled with `--no-bootstrap` flag.
+- Added: [#5485](https://github.com/ethereum/aleth/pull/5485) aleth-bootnode now by default connects to official Ethereum bootnodes. This can be disabled with `--no-bootstrap` flag.
```
I believe this also should be `Changed:` as it is not a new feature.

gumb0 replied:
Why not, this is new code and new flag added

chfast replied:
Can be. It rather feels to me we are changing only how features are enabled / disabled. But I don't care much.

halfalicious commented:
Rebased to address merge conflict

halfalicious commented:
> 
> 
> Also keep the list grouped by "kind": Added, Changed, Removed, Fixed, ...

Did that in the new version (grouped changes in that order: Added, Changed, Removed, Fixed)
